pymc3/stats.py

```diff
@@ -342,7 +342,7 @@ def bpic(trace, model=None):
 
 
 def compare(traces, models, ic='WAIC', method='stacking', b_samples=1000,
-            alpha=1, seed=None):
+            alpha=1, seed=None, round_to=2):
```

Comment:
I dont have a very strong view - but if we are going to implement the rounding, I would increase the default to 3.